src/plugins/source.js

```diff
@@ -1,11 +1,12 @@
 import path from 'path'
 import fs from 'fs'
+import date from 'date-and-time'
 
 export default async function () {
   const content = await this.driver.getSource()
 
   return new Promise((resolve) => {
-    const pathToLog = path.resolve(process.cwd(), 'appium_source.xml')
+    const pathToLog = path.resolve(process.cwd(), 'appium_source_'+date.format(new Date(), 'YYYYMMDD[T]HHmmss')+'.xml')
```

Comment:
1) Sorry, I misled you. We should be able to put the source file with strong naming. We can solve it with some additional param.
```
export default async function ({ strongSourceFileName = true }) {
```
You can call `await driver.source({ strongSourceFileName: false })` to put unique source files.
2) Let's rename the const to something like `sourceFilePath` because it's not logs, it's a source of the page.
3) Don't need to include an external library for one simple action. Use native `Date`